### DISCUSSION TITLE
save_crops into tif file format

### DISCUSSION THREAD

**Discussion by Ramanmagar** [azavea/raster-vision#1736] [Q&A]
I have model.pth...if i predicted some boxes with tif file .than want to save  save_crops into .tif file format for all individual crops.

Thank You

**Comment by AdeelH** [comment]
Your question is not clear.

**Reply by Ramanmagar** [reply]
I have model of Semantic Segmentation: ISPRS Potsdam, now i want to test one tif file ..after test which result will come like crop images that images want to save in tif format.

Thank you

**Reply by Ramanmagar** [reply]
This is possible sir
